### PR TITLE
feat : Added Alert

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

File: docs/Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.bat-alert-error {
+    background-color: var(--alert-error-bg);
+    color: var(--alert-error);
+}
+
+.bat-alert-dismiss {
+    display: flex;
+    justify-content: space-between;
+
```
Comment (junaidshaikh-js) [inline docs/Components/alert.css]: There shouldn't be any empty lines in the code. Also, be consistent in the spacing between two blocks

File: docs/css/documentation.css [hunk 2]
```diff
@@ -18,7 +18,8 @@
 .bat-sidebar {
     top: 0;
     position: sticky;
-
+    height: 90vh;
+    overflow: scroll;
     /* background-color: var(--prussian-blue); */
```
Comment (junaidshaikh-js) [inline docs/css/documentation.css]: commented code should be removed

File: docs/get-started.html [hunk 3]
```diff
@@ -22,10 +22,12 @@
...
       <div class="bat-nav">
         <nav>
-          <div class="bat-logo"><i class="fad fa-bat"></i> BatstrapUI</div>
+          <a href="index.html">
+            <div class="bat-logo"><i class="fad fa-bat"></i> BatstrapUI</div></a
+          >
           <div class="bat-menu">
             <ul id="bat-menu-list">
-              <li><a href="index.html">Home</a></li>
+              <a href="index.html"><li>Home</li></a>
```
Comment (junaidshaikh-js) [inline docs/get-started.html]: `a` tag should be nested inside `li` element.

**Comment by saadahmadhere** [comment]
Looks good!

File: docs/Components/alert.css (hunk 1 above)
Comment (ZuberDunge) [reply docs/Components/alert.css]: Thank you for your review, I've updated the code.

**Pull request merged**